.flake8

```diff
@@ -1,19 +1,59 @@
 [flake8]
-# E203 skipped because it annoys black.
-# no pyproject.toml support https://gitlab.com/pycqa/flake8/-/issues/428
-show-source = true
-extend-ignore =
-  E203,
-  F401,
-max-line-length = 100
+
 builtins = _
+
+# Print the total number of errors:
+count = true
+
+# Don't even try to analyze these:
 extend-exclude =
-  # skip tool cache dirs
+  # No need to traverse egg info dir
+  *.egg-info,
+  # tool cache dirs
   *_cache
-  # skip project env vars
+  # project env vars
   .env,
-  # skip adjacent venv
+  # GitHub configs
+  .github,
+  # Cache files of MyPy
+  .mypy_cache,
+  # Cache files of pytest
+  .pytest_cache,
+  # Temp dir of pytest-testmon
+  .tmontmp,
+  # Occasional virtualenv dir
+  .venv
+  # VS Code
+  .vscode,
+  # Temporary build dir
+  build,
+  # This contains sdists and wheels of ansible-navigator that we don't want to check
+  dist,
+  # Metadata of `pip wheel` cmd is autogenerated
+  pip-wheel-metadata,
+  # adjacent venv
   venv
+
+# IMPORTANT: avoid using ignore option, always use extend-ignore instead
+# Completely and unconditionally ignore the following errors:
+extend-ignore =
+  E203,  # skipped because it annoys black
+  F401,  # skipped for unknown reason
+  WPS300,  # "Found local folder import" -- nothing bad about this
+  WPS306,  # "Found class without a base class: *" -- irrelevant under Python 3
+  WPS422,  # "Found future import: *" -- we need these for multipython
```

Comment:
```
Forbid __future__ imports.

Reasoning:
Almost all __future__ imports are legacy python2 compatibility tools that are no longer required.

Solution:
Remove them. Drop python2 support.
```

The only place we violate this is and probably will be the tests fixtures directory because it may contain ansible content which was/is subject to a different rule set.  Can we limit this to just test/fixtures with an explanation?

Reply:
Not necessarily. There are some of the future imports that are related to newer pythons. Like `from __future__ import annotations`.

Reply:
Oh, wait, there's a list of allowed ones: https://wemake-python-stylegui.de/en/latest/pages/api/constants.html#wemake_python_styleguide.constants.FUTURE_IMPORTS_WHITELIST.

Reply:
In general, I'm in favor of only ignoring this in the test fixtures. The problem is that I've seen flake8 not merging several ignores that are listed in specific file paths and matching wildcards. Basically, if you have two entries like `tests/test_file.py` and `tests/test_*.py` only one will be taken into account by flake8. Not sure if it's a bug or by design but I remember this confused me and I couldn't find a proper solution for the problem.

Reply:
No experience there, if the change is outside the scope of this PR can we log an issue to resolve with a follow up PR?

Reply:
I'll just drop this for now.